distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1871,6 +1871,86 @@ type Table
             new_column = column_to_parse.parse type format on_problems
             table.set new_column new_name=column_to_parse.name set_mode=Set_Mode.Update
 
+    ## GROUP Standard.Base.Conversions
+       Formats `Column`s within a `Table` using a format string,
+       `Date_Time_Formatter`, or `Column` of format strings.
+
+       Arguments:
+       - columns: The columns to format. If not specified, all columns will be
+         formatted. The columns can have different types, but all columns must
+         be compatible with any provided `format` value.
+       - format: The type-dependent format string to use to format the values.
+         If `format` is `""` or `Nothing`, .to_text is used to format the value.
+         In case of date/time columns, the format can also be a
+         `Date_Time_Formatter`. If `format` is a `Column`, it must be a text
+         column.
+       - locale: The locale in which the format should be interpreted.
+         If a `Date_Time_Formatter` is provided for `format` and the `locale` is
+         set to anything else than `Locale.default`, then that locale will
+         override the formatters locale.
+       - error_on_missing_columns: if `True` (the default) raises an error if
+         any column is missing. Otherwise, reported as a problem.
+       - on_problems: Specifies how to handle if a problem occurs, raising as a
+         warning by default.
+
+       ! Error Conditions
+
+         - If a column in `columns` is not in the input table, a
+           `Missing_Input_Columns` is raised as an error or problem
+           following the `error_on_missing_columns` rules.
+         - If a provided `format` value is not compatible with all selected
+           columns, and error will be thrown. The error will depend on the
+           mismatch between the format and the column.
```

Comment:
```suggestion
         - If a provided `format` value is not compatible with all selected
           columns, an error will be thrown. The error will depend on the
           mismatch between the format and the column.
```

What are the possible error types that may be thrown here? I think we should mention the names of the error types.

Reply:
Done